<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowAnalyzer</class>
 <widget class="QMainWindow" name="WindowAnalyzer">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>345</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>899</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>899</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Анализатор спектра</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/newPrefix/Spectro16.ico</normaloff>:/newPrefix/Spectro16.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>304</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cbFreq">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>18</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Диап f</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbU">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>62</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Диап U</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCh">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>65</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>104</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кан</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>133</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Курсоры</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>62</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>f1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>92</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>f2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leU0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>58</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>60</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>U1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leU1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>87</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>91</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>U2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDeltaFreq">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>118</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>df</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leFreq1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>87</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbPos1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>87</y>
       <width>18</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDeltaU">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>118</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="chbEnableCurs">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Включить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leUmax">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>148</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leFUmax">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>148</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>fumax</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>121</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>134</x>
       <y>150</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Umax</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leU_DC">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>22</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>U0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leFreq0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbPos0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>111</x>
       <y>60</y>
       <width>18</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="cbPort">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>17</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbWindow">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Окно Хэмминга</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Окно Блэкмана</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прямоугольное</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pbUpdate">
    <property name="geometry">
     <rect>
      <x>831</x>
      <y>16</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>Цвет</string>
     </property>
     <addaction name="aColorBackground"/>
     <addaction name="aColorGrid"/>
     <addaction name="aColorSignal"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="aCalibrate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="aColorBackground">
   <property name="text">
    <string>Фон</string>
   </property>
  </action>
  <action name="aColorGrid">
   <property name="text">
    <string>Сетка</string>
   </property>
  </action>
  <action name="aColorSignal">
   <property name="text">
    <string>Сигнал</string>
   </property>
  </action>
  <action name="aCalibrate">
   <property name="text">
    <string>Калибровать</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
